src/utils/emailLink.jsx

```diff
@@ -0,0 +1,48 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import A from '../atoms/a';
+
+const propTypes = {
+    className: PropTypes.string,
+    email: PropTypes.string,
+    tabIndex: PropTypes.number,
+};
+
+const defaultProps = {
+    className: undefined,
+    email: undefined,
+    tabIndex: -1,
+};
+
+function EmailLink(props) {
+    const {
+        className,
+        email,
+        tabIndex,
+    } = props;
+
+    if (!email) {
+        return null;
+    }
+
+    const onClick = (event) => {
+        event.stopPropagation();
+
+        window.location.href = `mailto:${email}`;
+    };
```

Comment:
Any reason to do this, versus actually use a plain-old regular `<a> with this `href`?

Reply:
I’ll think about it.

Reply:
Right now the A component is not an actual `<a>`.

Reply:
Switched to `<a>`, added `href` attribute, and remove the `onClick` method.

Reply:
Fixed as requested in ba0497c.  Resolving.